aten/src/ATen/native/native_functions.yaml

```diff
@@ -271,12 +271,12 @@
     CPU, CUDA: angle_out
 
 - func: view_as_real(Tensor(a) self) -> Tensor(a)
-  variants: function
+  variants: function, method
```

Comment:
cc @anjali411 there were some good reasons not to have these as methods?

Reply:
There wasn't a specific reason why we didn't wanna add them as methods. I think it was more like that they were added as utility functions to help people migrate to the new complex types and should be migrated at some point. 

@bdhirsh is there a reason why you want to make `view_as_real` and `view_as_complex` available as methods?

Reply:
Yeah I should have called this out more explicitly.

It simplifies the codegen a little, because those are currently the only two view ops that aren't methods.


Let me know if you'd rather not expand the api surface, and I can update the codegen